### PR TITLE
Update TS SDK telemetry options examples

### PR DESCRIPTION
**Pull request by mjameswh**
## What does this PR do?

- Updated examples of how to use `telemetryOptions: { ... }`. See this [PR](https://github.com/temporalio/sdk-typescript/pull/963).

### CODE REVIEW

File: docs-src/typescript/how-to-emit-metrics-typescript.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ telemetryOptions: {
     metrics: {
       prometheus: { bindAddress: '0.0.0.0:9464' },
     },
-    logging: { forward: { level: 'DEBUG' } },
+    logging: {
+      forward: {},
```
Comment (lorensr) [inline docs-src/typescript/how-to-emit-metrics-typescript.md:25]: what does the empty object do?

File: docs/application-development/observability.md [hunk 2]
```diff
@@ -884,7 +887,7 @@ map[string]interface{}{
 </TabItem>
 <TabItem value="java">
 
-In your Workflow code, call the [`upsertSearchAttributes(Map<String, ?> searchAttributes)`](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/workflow/Workflow.html#upsertSearchAttributes(java.util.Map)) method.
+In your Workflow code, call the [`upsertSearchAttributes(Map<String, ?> searchAttributes)`](<https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/workflow/Workflow.html#upsertSearchAttributes(java.util.Map)>) method.
```
Comment (lorensr) [inline docs/application-development/observability.md:890]: intentional?

File: docs-src/typescript/how-to-emit-metrics-typescript.md (hunk 1 above)
Comment (mjameswh) [reply docs-src/typescript/how-to-emit-metrics-typescript.md:25]: The empty object enables the option. That's unfortunate, but was already established previously, and we ended up deciding not to change this at this time.

Comment (lorensr) [reply docs-src/typescript/how-to-emit-metrics-typescript.md:25]:
```suggestion
      forward: {}, // this enables log forwarding
```

**Review by lorensr — APPROVED** [review]

**Comment by lorensr** [comment]
Format check failing

image

**Comment by axfelix** [comment]
@mjameswh Would be happy to merge this if it gets picked up again!

File: docs/application-development/observability.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ telemetryOptions: {
     },
     logging: {
       forward: {},
-      filter: makeTelemetryFilterString({ core: 'INFO', other: 'WARN' })
+      filter: makeTelemetryFilterString({ core: 'INFO' })
```
Comment (flossypurse) [inline docs/application-development/observability.md:143]: this file should no longer exist -

**Comment by mjameswh** [comment]
Let's close this PR. I will restart from scratch.

**Pull request closed without merging**